.circleci/config.yml

```diff
@@ -116,6 +118,7 @@ jobs:
       - run: sudo make phpthemis_install
       - run: sudo bash -c 'echo "extension=phpthemis.so" > /etc/php/7.0/cli/conf.d/20-phpthemis.ini'
       - run: sudo make jsthemis_install
+      - run: make rustthemis_examples_install
```

Comment:
maybe `rustthemis_examples_install` should be a part of `prepare_tests_all` makefile target?
https://github.com/cossacklabs/themis/blob/master/tests/test.mk#L63

Reply:
I thought about that initially because it really makes sense at the first. But then I saw that this target is [used by PHP builds](https://github.com/cossacklabs/themis/blob/0d5ea505c23b4ecf786ef98d349765bf41d46a88/.circleci/config.yml#L139) and decided to not risk adding anything there.

I have to admit that this hack with `cargo install` is ugly. It installs binaries into user's home directory so that's at least one reason to make sure that this target cannot be executed accidentally. That's why I'd prefer to keep this target standalone, at least if it's going to remain as it is now. I wouldn't like the developers to accidentally execute it and end up with some unexpected `keygen_tool` in their PATH.

I'll see if I can persuade Cargo to install the tools somewhere in `./$(BIN_PATH)` (like other languages in `prepare_tests_all` do). That should make this target useable not only in CI environment.

Would it be okay to call `prepare_tests_all` from the integration test suite? This target seems to install files for unit-testing the wrappers, not the integration ones.

Reply:
You're right, `prepare_tests_all` is called to install the unit tests dependencies for manual / CI testing. 

> Would it be okay to call prepare_tests_all from the integration test suite?
Probably it's a bad idea.

I'm concerned if main Makefile it's a good place to install rust examples from
Please see comment below.

https://github.com/cossacklabs/themis/pull/356#pullrequestreview-196836383